luarules/gadgets.lua

```diff
@@ -1662,14 +1662,21 @@ function gadgetHandler:TerraformComplete(unitID, unitDefID, unitTeam,
 end
 
 function gadgetHandler:AllowWeaponTargetCheck(attackerID, attackerWeaponNum, attackerWeaponDefID)
-	for _, g in ipairs(self.AllowWeaponTargetCheckList) do
-		if not g:AllowWeaponTargetCheck(attackerID, attackerWeaponNum, attackerWeaponDefID) then
-			return false
+local ignore = true
+for _, g in ipairs(self.AllowWeaponTargetCheckList) do
+	local allowCheck, ignoreCheck = g:AllowWeaponTargetCheck(attackerID, attackerWeaponNum, attackerWeaponDefID)
+	if not ignoreCheck then
```

Comment:
spacing here looking bad

Reply:
I made a separate PR and @'d you there, check your notifications. There's a screenshot showing how it looks when github isn't lying about my tabbing ;p